go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/multiformats/go-multihash v0.1.0
 	github.com/onflow/atree v0.1.1
 	github.com/onflow/cadence v0.21.0
-	github.com/onflow/flow v0.2.3-0.20220124225841-6dd3bed23faf
+	github.com/onflow/flow v0.2.3-0.20220131193101-d4e2ca43a621
```

Comment:
[the change to open API spec](https://github.com/onflow/flow/pull/790) which removes the `signer_index` has already been merged in.